CHANGELOG.md

```diff
@@ -250,6 +250,7 @@
 * Avoid copying copy-on-write data structures when the write does not actually
   change the existing value.
 * Improve performance of deleting all rows in a TableView.
+* Added `LangBindHelper::get_backlink_table()` to help bindings to follow backlinks.
```

Comment:
This is not the correct line. 😉 Please add a link to this PR as well (as can be seen in the entries at the top).